recipes/systemc/meta.yaml

```
@@ -0,0 +1,119 @@
+          - test -f "$PREFIX"/lib/libsystemc-{{ version }}"$SHLIB_EXT"  # [not win]
+    - name: libsystemc-static                                    # [not win]
+      files:                                                     # [not win]
+          - lib/libsystemc.a                                     # [not win]
+    - name: systemc-doc                                          # [not win]
+      files:                                                     # [not win]
+          - share/doc/systemc/                                   # [not win]
+    - name: systemc-dev                                          # [not win]
+      requirements:                                              # [not win]
+          build:                                                 # [not win]
+              - {{ compiler('cxx') }}                            # [not win]
+          run:                                                   # [not win]
+              - {{ pin_subpackage('libsystemc', exact=True) }}   # [not win]
+      files:                                                     # [not win]
+          - lib/libsystemc.so                                    # [not win]
+          - lib/pkgconfig/systemc.pc                             # [not win]
+          - lib/pkgconfig/tlm.pc                                 # [not win]
+          - include/sysc                                         # [not win]
+          - include/systemc*                                     # [not win]
+          - include/tlm*                                         # [not win]
+          - include/tlm_core                                     # [not win]
+          - include/tlm_utils                                    # [not win]
+          - etc/conda/*activate.d/*activate-systemc.sh           # [not win]
+      test:                                                      # [not win]
+        requires:                                                # [not win]
+          - {{ compiler('cxx') }}                                # [not win]
+          - {{ pin_subpackage('systemc-doc', exact=True) }}      # [not win]
+          - make                                                 # [not win]
+          - sed                                                  # [not win]
+        commands:                                                # [not win]
+          - test "$SYSTEMC_HOME" == "$PREFIX"                    # [not win]
+          - cp -r "$PREFIX"/share/doc/systemc/examples .         # [not win]
+          # patch Makefile.rules because we want to use the installed libsystemc  
+          # AND we want to use the dynamic lib which the linker should be able to find
+          - "sed -i 's|$(EXE): $(OBJS) $(SYSTEMC_LIB_DIR)/libsystemc.a|$(EXE): $(OBJS)|' examples/build-unix/Makefile.rules"  # [not win]
+          - make -C examples/build-unix -j $CPU_COUNT run        # [not win]
+          - make -C examples/build-unix -j $CPU_COUNT check      # [not win]
```

```suggestion
          - make -C examples/build-unix -j $CPU_COUNT run        # [linux]
          - make -C examples/build-unix -j $CPU_COUNT check      # [linux]
          - make -C examples/build-unix -j $CPU_COUNT run LDFLAG_RPATH=-Wl,-rpath,        # [osx]
          - make -C examples/build-unix -j $CPU_COUNT check LDFLAG_RPATH=-Wl,-rpath,     # [osx]
```

`examples/build-unix/Makefile.rules` uses GNU syntax for the rpath option that doesn't work with the macos linker.  Need to use a comma instead of `=`.